content/articles/image-compression-using-PCA/index.md

```diff
@@ -0,0 +1,339 @@
+
+Here I have used <b>u<sub>1</sub></b> and <b>u<sub>2</sub></b> as a basis instead of <b>x</b> and <b>y</b>. We can observe that all the points have a minimal component in the direction of <b>u<sub>2</sub></b>(almost noise). It seems that the same data that was initially in R<sub>2</sub>:(x,y) can now be represented in R<sub>1</sub>:(u<sub>1</sub>) by making an intelligent choice of basis.
+
+#### Why not care about u<sub>2</sub>.
+
+Because the variance in the data in this direction is minimal (all data points have almost the same value in the u<sub>2</sub> direction), if we were to build a classifier on top of this data, then u<sub>2</sub> would not contribute to the classifier as the points are not distinguishable along this direction. In this way, we have reduced the dimensionality.
+
+> In general, we are interested in representing the data using fewer dimensions such that the data has higher variance along these dimensions.
+
+#### Correlation
+
+Data correlation is how one set of data may correspond to another set. If two columns are highly correlated(or have high covariance), one of them is redundant since it is linearly dependent on the other column. We can normalize the correlation to get the correlation coefficient. The formula for the correlation coefficient is defined as :
+
+<center>
+
+
+![Equation](/engineering-education/image-compression-using-PCA/capture.jpg)
+
+</center>
+
+#### Requirements for Dimensionality Reduction
+
+In general, we are interested in representing the data using fewer dimensions such that,
+
+- The data has <b>high variance</b> along these dimensions.
+- The dimensions are linearly <b>independent</b>(uncorrelated)
+- If we want to reduce dimensions by transforming the data into a new basis, the resulting basis should be <b>orthogonal</b>.
+
+### Principal Component Analysis(PCA)
+
+> **Principal Component Analysis (PCA)** is a statistical procedure that uses an orthogonal transformation that converts a set of correlated variables to a set of uncorrelated variables.
+
+To explain the concept of PCA mathematically, I will take an example:
+
+#### Orthogonal Transformation
+
+##### Assumptions 
+
+Let p<sub>1</sub>, p<sub>2</sub>, p<sub>3</sub>,....,p<sub>n</sub> be a set of n orthonormal vectors. Let p be a square matrix of order n such that p<sub>1</sub>, p<sub>2</sub>, p<sub>3</sub>,....,p<sub>n</sub> are the columns of matrix p.
+
+Let x<sub>1</sub>, x<sub>2</sub>, x<sub>3</sub>,.....,x<sub>n</sub> &isin; R<sup>n</sup> be n data points and let X be a square matrix such that x<sub>1</sub>, x<sub>2</sub>, x<sub>3</sub>,.....,x<sub>n</sub>  are the rows of a matrix (Assumption: Data is zero-mean and unit variance), why we have this assumption we will come to this at later part.
+
+##### Transformation
+
+Suppose we want to represent x<sub>i</sub> using the new basis P.
+
+`x<sub>i</sub> = &prop;<sub>i1</sub>p<sub>1</sub> + &prop;<sub>i2</sub>p<sub>2</sub> + ....... + &prop;<sub>in</sub>p<sub>n</sub>`
```

Comment:
This is how this equation is in preview. 
![image](https://user-images.githubusercontent.com/26842636/130475012-29e4bb51-c964-4a5f-8cc4-2c7ab8085e5a.png)

You will have to use in-line Latex for all the equations.